api/rpc/stat/stat.go

```diff
@@ -29,9 +29,16 @@ func New(cfg Config) (*Stat, error) {
 	return &Stat{conn: c}, err
 }
 
-// ExecuteQuery implements business logic for StateServer interface
+// ExecuteQuery implements business logic for StatServer interface
```

Comment:
I didn't mean this is a typo. I meant literally is this statement accurate? I don't think we have a StatServer interface. Am I missing it?